src/client.ts

```diff
@@ -3339,6 +3340,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
 
     /**
      * Disable backing up of keys.
+     * @deprecated There is no public API to disabled backup, delete it if needed
```

Comment:
If there is a trusted backup it will be used (to backup keys), there is no additional on/off. I am not sure to find a use case for an additional api to enable/disable?
Anyhow atually the way to disable is to delete any backup.